examples/ec2/spotfleetrequest.yaml

```
@@ -0,0 +1,67 @@
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: SpotFleetRequest
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ec2/v1beta1/spotfleetrequest
+  labels:
+    testing.upbound.io/example-name: cheap_compute
+  name: cheap-compute
+spec:
+  forProvider:
+    allocationStrategy: capacityOptimized
+    iamFleetRole: arn:aws:iam::${data.aws_account}:role/aws-service-role/spotfleet.amazonaws.com/AWSServiceRoleForEC2SpotFleet
```

For Account ID we need to use the following key for reading the value from the data source: `${data.aws_account_id}`

```suggestion
    iamFleetRole: arn:aws:iam::${data.aws_account_id}:role/aws-service-role/spotfleet.amazonaws.com/AWSServiceRoleForEC2SpotFleet
```